package.json

```diff
@@ -29,7 +29,8 @@
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "^3.4.3",
-    "react-scroll": "^1.8.1"
+    "react-scroll": "^1.8.1",
+    "typescript": "^4.2.3"
```

Comment:
Remove typescript. 

What is react-scroll? why do we need it?

Reply:
I have no idea what react-scroll is, but I didn't add it. To be on the safe side, I don't want to delete it in this PR.